SpatialStructure-7/readme.md

```diff
@@ -0,0 +1,35 @@
+
+### Intent
+
+An IFC-based test case for the HNTB storyline "prestressed concrete bridge".
+
+The model was created using Autodesk InfraWorks and refined in Autodesk Revit. 
+Inside Revit, a prototypical Dynamo interface transfers the model into appropriate IFC representations. 
+
+The current IFC model covers aspects of spatial decomposition. 
+
+
+### Prerequisites
+
+This scenario builds upon following other scenarios:
+
+- Project Setup
+- Spatial Structure 
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+
+- SpatialStructureElements such as `IfcBridge`, `IfcRoad`, and `IfcFacilityPart` with appropiate PDTs
+- `IfcElement` subclasses to describe the bridge objects 
+
```

Comment:
Would it be worth just adding a Note (caveat) here that:

> "the road elements are not semantically defined and structured correctly according to IFC best practice and are only included for geometric illistration."

or update the file to be correct in terms of pavement and surface markings?

Reply:
tried to resolve this commit with 31d06a8731f6ff3957b6ab579f09f2b4d8d709db